docs/source/recipes/lora_finetune_single_device.rst

```diff
@@ -0,0 +1,60 @@
+.. _lora_finetune_recipe_label:
+
+=============================
+LoRA Single Device Finetuning
+=============================
+
+This recipe supports finetuning on next-token prediction tasks using `LoRA <https://arxiv.org/abs/2106.09685>`_,
```

Comment:
I like this idea!

I wonder if it might be better to include in the recipe overviews? Almost like a table of different compute desiderata and which recipes fulfill them.